homeassistant/components/zwave_js/__init__.py

```diff
@@ -114,22 +118,32 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     dev_reg = device_registry.async_get(hass)
     ent_reg = entity_registry.async_get(hass)
 
+    registered_unique_ids: dict[str, dict[str, set[str]]] = {}
```

Comment:
Instead of calling `setdefault`, just use a `defaultdict` :)

Reply:
thanks for the tip, never used it before but a good idea to keep in the back pocket